sdk/keyvault/internal/challenge_policy.go

```diff
@@ -1,5 +1,5 @@
-//go:build go1.16
-// +build go1.16
+//go:build go1.18
+// +build go1.18
```

Comment:
The `+build` tags are only needed for Go < 1.17, so they're redundant now.

Reply:
I'll update the code generator with this change shortly.

Reply:
I will point out though that removing them will break building with older versions of Go.

Reply:
This uses the `to.Ptr` method, it won't build with older versions of Go anyways.

Reply:
To clarify, I'm talking about track 1.  We had this problem in the past, see https://github.com/Azure/azure-sdk-for-go/issues/6846.  We might need to keep these until track 1 is out of `main` (needs investigation).

Reply:
I'll keep these for now